README.md

```diff
@@ -97,267 +112,12 @@ In general, always refer to this page for developer information. If you think
-the following shorthand can also be used:
-
-```sh
-make benchmark VARIABLE="value" ...
-```
-
-A combination of the above approaches is also possible (e.g. it may be useful to
-`export` the `SYSTEM_NAME` variable, and specify the others at every benchmark
-run).
-
-Supported environment variables are described in the following list:
-
-*   `BENCHMARK={spmv, solver, preconditioner}` - The benchmark set to run.
-    Default is `spmv`.
-    *   `spmv` - Runs the sparse matrix-vector product benchmarks on the
-                 SuiteSparse collection.
-    *   `solver` - Runs the solver benchmarks on the SuiteSparse collection.
-                The matrix format is determined by running the `spmv` benchmarks
-                first, and using the fastest format determined by that
-                benchmark. The maximum number of iterations for the iterative
-                solvers is set to 10,000 and the requested residual reduction
-                factor to 1e-6.
-    *   `preconditioner` - Runs the preconditioner benchmarks on artificially
-                generated block-diagonal matrices.
-*   `DRY_RUN={true, false}` - If set to `true`, prepares the system for the
-    benchmark runs (downloads the collections, creates the result structure,
-    etc.) and outputs the list of commands that would normally be run, but does
-    not run the benchmarks themselves. Default is `false`.
-*   `EXECUTOR={reference,cuda,omp}` - The executor used for running the
-    benchmarks. Default is `cuda`.
-*   `SEGMENTS=<N>` - Splits the benchmark suite into `<N>` segments. This option
-    is useful for running the benchmarks on an HPC system with a batch
-    scheduler, as it enables partitioning of the benchmark suite and running it
-    concurrently on multiple nodes of the system. If specified, `SEGMENT_ID`
-    also has to be set. Default is `1`.
-*   `SEGMENT_ID=<I>` - used in combination with the `SEGMENTS` variable. `<I>`
-    should be an integer between 1 and `<N>`. If specified, only the `<I>`-th
-    segment of the benchmark suite will be run. Default is `1`.
-*   `SYSTEM_NAME=<name>` - the name of the system where the benchmarks are being
-    run. This option only changes the directory where the benchmark results are
-    stored. It can be used to avoid overwriting the benchmarks if multiple
-    systems share the same filesystem, or when copying the results between
-    systems. Default is `unknown`.
-
-Once `make benchmark` completes, the results can be found in
-`<Ginkgo build directory>/benchmark/results/${SYSTEM_NAME}/`. The files are
-written in the JSON format, and can be analyzed using any of the data
-analysis tools that support JSON. Alternatively, they can be uploaded to an
-online repository, and analyzed using Ginkgo's free web tool
-[Ginkgo Performance Explorer (GPE)](https://ginkgo-project.github.io/gpe/).
-(Make sure to change the "Performance data URL" to your repository if using
-GPE.)
```

Comment:
Actually, where did the benchmarking section go? I would also put it in its own `BENCHMARKING.md`

Reply:
Sorry, deleted it by mistake. Added it now.